src/java/org/apache/cassandra/utils/FBUtilities.java

```diff
@@ -921,6 +911,16 @@ public static String prettyPrintAverage(double average)
         return String.format("%.2f", average);
     }
 
+    /**
+     * Convert the given size in bytes to a human-readable value using binary (i.e. 2^10-based) modifiers.
+     * For example, 1.000KiB, 2.100GiB etc., up to 8.000 EiB.
+     * @param size      Number to convert.
+     */
+    public static String prettyPrintMemory(long size)
```

Comment:
This method has been moved closer to `prettyPrintMemory(long size, String separator)`